server/config.ts

```diff
@@ -130,6 +130,7 @@ export default {
     },
   },
   googleAnalyticsMeasurementId: get('GOOGLE_ANALYTICS_MEASUREMENT_ID', ''),
+  teamEmail: get('TEAM_EMAIL', 'info@digital.justice.gov.uk'),
```

Comment:
I'd default to our team email (`non-associations-team@digital.justice.gov.uk`) and not require set it explicitly, unless you think this is that sensitive (even tho, at this point it's public really)